src/erasure-code/jerasure/ErasureCodeJerasure.cc

```diff
@@ -158,10 +158,21 @@ bool ErasureCodeJerasure::is_prime(int value)
     151,157,163,167,173,179,
     181,191,193,197,199,211,223,227,229,233,239,241,251,257
   };
-  int i;
-  for (i = 0; i < 55; i++)
-    if (value == prime55[i])
+  int i = sizeof(prime55)/sizeof(int) -1;
```

Comment:
In C++ STL header file bits/stl_algo.h, std::binary_search used lower_bound to loockup value, so we can use std::binary_search here.